diff --git a/src/init.cpp b/src/init.cpp
index bbd24026b..a8531c472 100755
--- a/src/init.cpp
+++ b/src/init.cpp
@@ -947,8 +947,7 @@ bool AppInit2(ThreadHandlerPtr threads)
 
             // Now that we are forked we can request a shutdown so the parent
             // exits while the child lives on.
-            StartShutdown();
-            return true;
+            exit(0);
         }
 
         pid_t sid = setsid();
